### PR TITLE
Feat: Log Patterns - Major update of beta documentation

### PR DESCRIPTION
**Pull request by mculmone**
### Give us some context

* What problems does this PR solve?
The beta Patterns docs have been updated. All beta references removed, GA screenshots included, language updated to reflect the current state of the release.
* Add any context that will help us review your changes such as testing notes,
I had some trouble with Yarn halfway through so I don't know if this runs quite correctly. Also there is one thing I have not changed; the URL of the page. It is currently : /docs/logs/log-management/ui-data/log-patterns-beta

It should be changed to: /docs/logs/log-management/ui-data/log-patterns

The redirect metadata should be updated to reflect this, and any of the main docs navigation that needs to be aware should be updated.
* If your issue relates to an existing GitHub issue, please link to it.

### Are you making a change to site code?

If you're changing site code (rather than the content of a doc), please follow
[conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

### CODE REVIEW

File: src/content/docs/logs/log-management/ui-data/find-unusual-logs-log-patterns-beta.mdx [hunk 1]
```diff
@@ -1,50 +1,45 @@
...
-  **[one.newrelic.com](https://one.newrelic.com) > Log management > Log patterns:** New Relic's log patterns feature automatically groups patterns so that you can focus on patterns in unusual or uncategorized logs.
-</figcaption>
-
-## Log patterns beta [#beta]
+* Orient more quickly through millions of logs.
+* Reduce the time it takes to identify unusual behavior in your log estate.
+* Monitor the frequency of known patterns over time to focus your energy on what matters, and exclude what's irrelevant.
 
-Our log patterns feature is currently in beta, and so its availability is limited to specific participating accounts. If you can't find the **Log patterns** button in your New Relic account's **Log management** UI, contact your account representative and ask to participate.
+Currently the patterns feature is opt-in; if you can't find the **Log patterns** button in your New Relic account's **Log management** UI, contact your account representative and ask to have it enabled. There will be up to a 24 hour lead time before the machine learning system generates a customized model for your account. Patterns will be enaabled by default for all customers at the end of May, 2021.
```
Comment (bmcfeely) [inline src/content/docs/logs/log-management/ui-data/find-unusual-logs-log-patterns-beta.mdx:32]: enaabled -> enabled

File: src/content/docs/logs/log-management/ui-data/find-unusual-logs-log-patterns-beta.mdx [hunk 2]
```diff
@@ -56,73 +51,49 @@ Our log patterns feature is currently in beta, and so its availability is limite
...
-    </tr>
-
-    <tr>
-      <td>
-        Events sampling
-      </td>
-
-      <td>
-        During the beta, the scope of the collected data required to train the patterns model is limited to 10M events per day. We are sampling your logging estate using NRQL filters such as level = `error`.
+        There is no separate pricing for Log Patterns. The only cost is for additional data generated and added to the log records. A pattern attribute will be added to all log that match a pattern. Attributes may also be added when common values such as GUIDs, IP Addresses, URL or email addresses are discovered and automatically extraced from the log message as prt of the pattern process.
```
Comment (bmcfeely) [inline src/content/docs/logs/log-management/ui-data/find-unusual-logs-log-patterns-beta.mdx:54]: all log -> all logs

Comment (bmcfeely) [inline src/content/docs/logs/log-management/ui-data/find-unusual-logs-log-patterns-beta.mdx:54]: Addresses -> addresses

Comment (bmcfeely) [inline src/content/docs/logs/log-management/ui-data/find-unusual-logs-log-patterns-beta.mdx:54]: prt -> part

**Comment by mculmone** [comment]
Fixing a few errors then will resubmit

**Pull request closed without merging**